Makefile

```diff
@@ -1,5 +1,20 @@
 CLANG_FORMAT := clang-format
-ALL_SRCS := $(wildcard *.cc) $(wildcard *.h)
+ALL_SRCS := $(shell \
+	find . -type f \
+		\( -path './build/*' -o -path './include/vendor/*' \) \
+		-prune -o \
+		\( -name '*.cpp' -o -name '*.h' \) \
+		-print \
+)
+
+ALL_LISTFILES := $(shell \
+	find . -type f \
+	-path './build/*' \
+	-prune -o \
+	\( -name 'CMakeLists.txt' -o -name '*.cmake' \) \
+	-print \
```

Comment:
Nit: This indentation is different from the one above it

Reply:
Blah. Will fix 😅